GNUmakefile

```diff
@@ -32,5 +32,3 @@ testacc: dev
 
 generate: install-packer-sdc
 	@go generate ./...
-	packer-sdc renderdocs -src ./docs -dst ./.docs -partials ./docs-partials
```

Comment:
Out of curiosity, why is this removed from `generate`? Additionally, if we remove this, should we also remove it from the `ci-release-docs` target as well?

Reply:
The removed command removes the docs preview that gets generated into the hidden .docs directory. We've seen how this creates noise, which can result preview files getting committed. I believe the .gitignore handles this already but I removed the command as it is not needed. 

The version of the command executed in `ci-release-docs` has slightly different paths as those files are used to generate the final docs.zip file needed for external plugin documentation ingestion.